.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```diff
@@ -0,0 +1,76 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
+  - type: textarea
+    id: reproduce-bug
+    attributes:
+      label: 'What steps are needed to reproduce the bug?'
+      placeholder: |
+        e.g. "CLI with `stylelint '**/*.css' --config myconfig.json`"
+    validations:
+      required: true
+  - type: input
+    id: stylelint-rule
+    attributes:
+      label: 'Which rule, if any, is the bug related to?'
+      placeholder: 'e.g. `number-leading-zero`'
+    validations:
+      required: false
+  - type: textarea
+    id: stylelint-configuration
+    attributes:
+      label: 'What Stylelint configuration is needed to reproduce the bug?'
+      placeholder: |
+        e.g.
+
+        ```json
+        {
+          "rules": {
+            "number-leading-zero": "always"
+          }
+        }
+        ```
+    validations:
+      required: false
+  - type: input
+    id: stylelint-version
+    attributes:
+      label: 'Which version of Stylelint are you using?'
+      placeholder: 'e.g. `14.0.0`'
+    validations:
+      required: false
+  - type: input
+    id: non-standard-syntax
+    attributes:
+      label: 'Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?'
+      placeholder: |
+        e.g. "Yes, it's related to SCSS nested properties."
+    validations:
+      required: false
+  - type: textarea
```

Comment:
Let's move `expected-behaviour` and `actual-behavior` to after `stylelint-version`.

But let's add a section after `stylelint-configuration` first:

- label: How do you run Stylelint?
- description: For example, CLI, Node,js API or PostCSS plugin.
- placeholder: e.g. "CLI with `stylelint "**/*.css" --config myconfig.json`"

So it goes:
- reproduce-bug
- stylelint-configuration
- run (new section)
- stylelint-version
- expected-behaviour
- actual-behaviour
- non-standard-syntax

So it follows the normal flow of:
- steps
- expected
- actual

But we're pulling out and specifically asking for the config, how it's being run and Stylelint version from the steps.

The final question, `non-standard-syntax`, is there to prepare users for when we say that Stylelint is designed for CSS, and the suggestion fixed well either be:
- we update our heuristics to ignore the syntax in Stylelint
- suggest they raise an issue in a 3rd party config or custom syntax

Reply:
Sounds good! Fixed: fd3eb80b, 1755f793